src/nina_vesseltun_ex/ex05/walker_sim.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Nina Mariann Vesseltun'
+__email__ = 'nive@nmbu.no'
+
+
+from random import randint
+from random import seed
+
+
+class Walker:
+    def __init__(self, x0, home):
+        self.x = x0
+        self.h = home
+        self.steps = 0
+
+    def move(self):
+        if randint(0, 1):
+            self.x += 1
+        else:
+            self.x -= 1
+        self.steps += 1
+
+    def is_at_home(self):
+        return self.x == self.h
+
+    def get_position(self):
+        return self.x
+
+    def get_steps(self):
+        return self.steps
+
+
+class Simulation:
+    def __init__(self, start, home, the_seed):
+        self.start = start
+        self.home = home
+        self.seed = the_seed
+
+    def single_walk(self):
+        pedestrian = Walker(self.start, self.home)
+        while not pedestrian.is_at_home():
+            pedestrian.move()
+        return pedestrian.get_steps()
+
+    def run_simulation(self, num_walks):
+        seed(self.seed)
```

Comment:
You should put this in `__init__` as random seed should be set once in the very first start.